src/services/localStorage.ts

```diff
@@ -248,6 +233,36 @@ const deleteWalletConnectInfo = () => {
   localStorage.removeItem('walletconnect')
 }
 
+// TODO: Migrating old localStorage key. Can be removed after being deployed for a while (deployed on 17.12.2021)
+const migrateOldlocalStorageKey = () => {
+  if (!isSupported()) return
+  const alreadyMigrated = localStorage.getItem('routes')
+  if (alreadyMigrated) return
+
+  const oldRoutes = localStorage.getItem('activeRoute')
+  if (oldRoutes) {
+    localStorage.setItem('routes', oldRoutes)
+  }
+  localStorage.removeItem('activeRoute')
+}
+migrateOldlocalStorageKey()
+
+// TODO: Migrating and adding toolDetails. Can be removed after being deployed for a while (deployed on 20.05.2022)
+const migrateToolDetails = () => {
+  if (!isSupported()) return
+  const oldRoutes = readAllRoutes()
+  const updatedRoutes = oldRoutes.map((oldRoute) => {
+    oldRoute.steps.map((step) => {
+      step.toolDetails = { logoURI: '', name: step.tool, key: step.tool.split('-')[0] }
+      return step
+    })
```

Comment:
I think we should add a filter to only add `toolDetails` if they don't exist already.